src/libraries/System.Text.Json/tests/Utf8JsonWriterTests.cs

```diff
@@ -2982,6 +2982,16 @@ public void WritingTooLargeBase64Bytes(bool formatted, bool skipValidation)
                 Assert.Throws<ArgumentException>(() => jsonUtf8.WriteBase64StringValue(value.AsSpan(0, 125_000_001)));
             }
 
+            using (var jsonUtf8 = new Utf8JsonWriter(output, options))
+            {
+                Assert.Throws<ArgumentException>(() => jsonUtf8.WriteBase64String(value.AsSpan(0, 166_666_667), value.AsSpan(0, 1)));
+            }
+
+            using (var jsonUtf8 = new Utf8JsonWriter(output, options))
+            {
+                Assert.Throws<ArgumentException>(() => jsonUtf8.WriteBase64String(Encoding.UTF8.GetString(value).ToCharArray().AsSpan(0, 166_666_667), value.AsSpan(0, 1)));
```

Comment:
I'm not sure if this is ok, may cause issues in test environments which are already under memory pressure?
https://github.com/dotnet/runtime/blob/c726f40f9fd76fe4ed1f3f16f955bc75c740a996/src/libraries/System.Text.Json/tests/Utf8JsonWriterTests.cs#L2966-L2973

Reply:
Elsewhere in this file and some of the other Utf8Json*cs tests there are some other similarly huge arrays newed up. Eg., `WriteTooLargeArguments()`